merkle.h

```diff
@@ -0,0 +1,113 @@
+#include "winternitz.h"
+#include <stack>
+
+class Merkle {
+  public:
+    struct Signature {
+      std::vector<Data> wint; 
+      std::vector<Data> auth; 
```

Comment:
what does these vectors mean/represent?

Reply:
@l4l `std::vector<Data> wint` represents winternitz signature and `std::vector<Data> auth` represents authorization path.

Reply:
Do you mind to add those comments directly in the code?